__tests__/lib/cli/envAlias.js

```diff
@@ -193,11 +193,6 @@ describe( 'utils/cli/envAlias', () => {
 			const result = envAlias.parseEnvAliasFromArgv( input.argv );
 
 			expect( result ).toEqual( input.expected );
-
-			// If there was an alias, should have parsed it
-			if ( input.argv.includes( '@app.env' ) ) {
-				expect( parseEnvAliasSpy ).toHaveBeenCalledWith( '@app.env' );
-			}
```

Comment:
We cannot test this reliably :-( The old code relied upon `module.exports`, which is gone now. However, tests for `parseEnvAlias` already cover this case.